bundled/README.md

```diff
@@ -1,49 +1,38 @@
 This folder contains third party projects bundled with deal.II
 
-PLEASE NOTE THAT THESE PROJECTS ARE COPYRIGHTED BY OTHERS THAN THE deal.II
+**PLEASE NOTE THAT THESE PROJECTS ARE COPYRIGHTED BY OTHERS THAN THE deal.II
 AUTHORS, but are included by permission. For details, consult the stated
-licenses below.
+licenses below.**
 
 Below is a detailed list of the content of each subdirectory, and the
 licenses that apply.
 
 
-./boost-1.49.0
-============
+./boost-1.56.0
```

Comment:
Also, best not to list individual version numbers given that they change over time. For example, below you still list 1.49.